### DISCUSSION TITLE
Increase length of chat response

### DISCUSSION THREAD

**Discussion by themeaningofmeaning** [FlowiseAI/Flowise#1562] [Q&A]
I'm currently testing a a Conversation Retrieval QA chain and the AI's responses seem to maintain a limit of 450 words every time.  Is there anyway to increase the response length of the chat response?

Also, it most responses seem to be cutoff.

**Comment by dkindlund** [comment]
@themeaningofmeaning , it's likely because your load balancer has a default 30 second max timeout, which causes conversations to get cut short from Flowise.  Check out this issue for more details on how to fix:
https://github.com/FlowiseAI/Flowise/issues/1468#issuecomment-1876240902

**Reply by themeaningofmeaning** [reply]
I'm running locally via Docker at the moment so I shouldn't be running into that issue on my end.  I noticed that using OpenAI's LLMs do not cause this issue, but using any open source LLM from HuggingFace does (Mixtral, etc).  Maybe it is a timeout on HuggingFace's side.

**Reply by dkindlund** [reply]
Most likely that's the case, @themeaningofmeaning .  If it were a Flowise issue, then it should be consistently timing out on any sort of LLM equally -- regardless of which LLM you use.
